data/ocicrypt_config.json

```diff
@@ -11,6 +11,9 @@
     },
     "keyprovider3": {
       "native": "attestation_agent"
+    },
+    "keyprovider4": {
+      "ttrpc": "unix:///tmp/ttrpc.sock"
```

Comment:
Should here be `unix:///tmp/keyprovider.sock` as https://github.com/confidential-containers/attestation-agent/pull/124/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R95 defines?

Reply:
Thanks @Xynnn007 , this config is for unit tests, but it will be better to consistent with `attestation-agent` docs, fixed now.